packages/api/src/brevo-api/brevo-api.module.ts

```diff
@@ -1,11 +1,12 @@
+import { CacheModule } from "@nestjs/cache-manager";
 import { Module } from "@nestjs/common";
 
 import { ConfigModule } from "../config/config.module";
 import { BrevoApiCampaignsService } from "./brevo-api-campaigns.service";
 import { BrevoApiContactsService } from "./brevo-api-contact.service";
 
 @Module({
-    imports: [ConfigModule],
+    imports: [ConfigModule, CacheModule.register({ ttl: 1000 * 60 })],
```

Comment:
according to the docs, ttl is in seconds, see: https://docs.nestjs.com/techniques/caching#customize-caching

I think we said we want the cache to be 1 Minute, so I would define it as ttl: 60